akka-http-core/src/main/java/akka/http/javadsl/model/Uri.java

```diff
@@ -186,6 +192,13 @@ public static Uri create(String uri) {
     return new JavaUri(akka.http.scaladsl.model.Uri.apply(uri));
   }
 
+  /**
+   * Returns an Uri wrapping the scaladsl representation.
```

Comment:
don't explain the implementation details :-)
"Returns the Java DSL representation of an Scala DSL Uri" :)